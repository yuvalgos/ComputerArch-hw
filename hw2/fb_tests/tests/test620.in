w 0x7c650
r 0x7c680
r 0x7c654
w 0x7c6e8
w 0x7c6c0
r 0x7c694
r 0x7c660
r 0x7c680
r 0x7c6d8
w 0x7c6a8
w 0x7c6b8
r 0x7c6e8
r 0x7c6a8
w 0x7c6d0
r 0x7c67c
w 0x7c664
r 0x7c69c
w 0x7c668
w 0x7c6e4
r 0x7c6c8
w 0x7c6b8
r 0x7c688
w 0x7c654
w 0x7c69c
w 0x7c654
r 0x7c694
r 0x7c690
r 0x7c6bc
r 0x7c650
w 0x7c678
r 0x7c664
r 0x7c6ac
w 0x7c650
w 0x7c6e0
w 0x7c6a0
w 0x7c668
w 0x7c6a0
w 0x7c6bc
r 0x7c6e4
w 0x7c6e4
w 0x7c6b0
