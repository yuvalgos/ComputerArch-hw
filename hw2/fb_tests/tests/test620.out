L1miss=0.805 L2miss=0.545 AccTimeAvg=131.073
